release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -80,11 +80,18 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
+- Added IDL client generation support using non-Ballerina Interface Definition Language
```

Comment:
```suggestion
- Added support to generate graphQL client definitions using graphQL IDL(i.e. config) files
```

Reply:
Should be: `Added the support to generate GraphQL client definitions using GraphQL IDL(i.e., config) files`